PR to track changes

shafeeq opened the pull request:
- Remove unavailable qualification IDs
- Change method of checking sandboxmode
- Support registering hit without qualification
- Fix get-account-balance

shafeeq committed:
Multiple changes
- Remove unavailable qualification IDs
- Change method of checking sandboxmode
- Support registering hit without qualification

shafeeq committed:
Change qualifications schema and add tests

shafeeq committed:
Bump API version

shafeeq committed:
Parse vectors of integers/locales for use with 'In' opertor in qualification requirements

shafeeq committed:
Add some schema, bump dependenceies versions

shafeeq committed:
Register and create hit using new schema and builder

shafeeq committed:
Add schema and functiopns for HIT operations

shafeeq committed:
Add get-assignments-for-hit and approve-rejected-assignment

shafeeq committed:
Convert multiple notifcation types using convert-params

shafeeq committed:
Generic parser for xml responses, change send-and-parse to use it

shafeeq committed:
Refactor code

shafeeq committed:
Add operations based on qualifications

shafeeq committed:
Add worker specific operations

shafeeq committed:
Add misc operations

shafeeq committed:
Fix assignments.clj

shafeeq committed:
Remove tests

shafeeq committed:
Change old methods to params and schema style

shafeeq committed:
Make parsed xml result a vector for consistency

shafeeq committed:
Add some integration tests.
- Register HIT Type
- Create HIT
- Search HIT
- Disable HIT

shafeeq committed:
Bugfix: In hits.clj for get-hits-for-qualification-type

shafeeq committed:
Bugfix: Method name in misc.clj

shafeeq committed:
Remove old code

shafeeq committed:
Bugfix: Change schemas for requester statistics

shafeeq committed:
Add more tests and refactor

shafeeq committed:
Use nomad for config in test

shafeeq committed:
Refactor test and test_helpers

shafeeq committed:
Minor edits

shafeeq committed:
Bugfix: Convert params for SendTestEventNotification

shafeeq committed:
Edit Readme

shafeeq committed:
Bugfix: Wrong operation string and schema in qualifications.clj

shafeeq committed:
Use env varibles for config

shafeeq committed:
Setup travis

smnirven commented:
Are you interested in taking over maintaining this repo? I'm no longer able to put time into this repository.

Thanks!

shafeeq committed:
Cleanup

shafeeq commented:
@smnirven Sure, I'd like to maintain it :)

shafeeq committed:
Add review policies to schema

shafeeq committed:
Add some unit tests

shafeeq committed:
Change tests namespace

shafeeq committed:
Fix bug in ApproveAssignments schema

shafeeq committed:
Remove redundant assignment method in hits.clj

shafeeq committed:
Update README

shafeeq committed:
Fix typo in method name

shafeeq committed:
Bump version to 0.7.0

shafeeq committed:
Change project url

shafeeq committed:
Update readme with link to getting started guide

shafeeq committed:
Throw exception if access keys are missing

shafeeq committed:
Restore previous config setup after running tests

shafeeq committed:
Fix typos in README

shafeeq committed:
Cleanup
- Move out a util fn from test helpers
- Move test specific files into test/
- Rename some variables for clarity

shafeeq committed:
Add validations for qualification requirement

shafeeq committed:
Initial idea for requester fn

shafeeq committed:
Use single requester fn, remove old functions, opn details in maps

shafeeq committed:
Move qualification validation to qualifications namespace

shafeeq committed:
Move schemas to seperate namespaces

shafeeq committed:
Minor edit + Throw exception on undefined operation key to requester

shafeeq committed:
Update README

shafeeq committed:
Remove special notation for constants
